### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by dpackham** [BobRak/OpenHAB-Smartthings#17]
Does this binding allow you to see and control devices FROM the ST hub as well or only from the OH2 interface control ST devices?

installed this and on the ST app I can see the ST devices but not the OH2

**Comment by dpackham** [comment]
Done :)

**Comment by BobRak** [comment]
You can see the devices in both the Smartthings App and in the OpenHAB web page (or app). It you aren't seeing things in OH2 then there is an issue with the configuration.  Do you see the Smartthing binding being loaded in the OH2 logs? Have you configured OH2 using discovery or files? If you are using discovery you need to need to select the Smartthings Binding first.

**Comment by dpackham** [comment]
I am trying to see the OH2 devices (Insteon/x10) switches on SH first so I can control them from SmartApps.  when I add the OpenHabAppV2 from My Apps and I set the IP address of the OH2 server up I only see devices already connected to my SH and none of the OH2 devices.  

how can I troubleshoot this?  

I will try to setup OH2 to see the SH devices later.

**Comment by BobRak** [comment]
I'm not at all sure about what you are trying to do. I assume by SH you mean a Smart Home hub opposed to a Smartthings hub (ST). I can't help you with SH.

On ST you will be able to see your devices that are connected to the ST hub. And, you should be able to see those device on OH2 if the configuration is correct. But, you won't be able to see devices on OH2 on ST that are not part of ST. At this point integrating OH2 devices back to ST isn't supported and I don't know if it is even possible.

**Comment by dpackham** [comment]
Yes I mean ST.   

for Example.

OH2 has an Insteon Switch Called "MasterBedroom" connected directly to the server via a Insteon PLM.  I can turn on/off this light with the OH2 UI.

I want to see and control "MasterBedroom" from a ST rule or button.  and I am guessing this is what does not work at this point correct?

**Comment by BobRak** [comment]
Yes, it is not possible for the ST **binding** to monitor non-ST devices and send status updates to the ST hub. In general OH2 bindings are responsible for monitoring and controlling devices connected to OH2 directly or through a "hub" type device. One binding does not know what the other is doing. I'm guessing it is possible to do this in OH2 using a facility other than a binding. But, that was not the design goal of this binding. I don't ever see the ST binding of supporting this.

One way this MIGHT be achieved is:

1. - Create "simulated" devices in the ST hub.
2. - Connect those devices to OH2.
3. - Use the OH2 rules engine to monitor the "real" devices for status changes and then update the OH2/ST "simulated" device. This would cause the ST binding to update the device and change the ST "simulated" device.

Could you just write your rule in OH2 using the rules engine?

**Comment by dpackham** [comment]
OK I did see them when I have the OH2 Hue emulation working. ill write some rules :)  Thanks

**Comment by dpackham** [comment]
Just FYI using the Hue Emulation on OH2 i can see the insteon lights that are connected to the OH2 server and I can control them from the ST things interface.  on/off etc

**Comment by BobRak** [comment]
Very good to hear. Thanks for letting me know.
